README.md

```diff
@@ -118,6 +118,10 @@ Run the command:
 $ make acc-test
 ```
 
+### Test provider changes locally
+Please make use of a unique path as provided in the `Makefile` while building the provider with changes 
+and kindly use the same path in the source while using the provider to test the local changes.
+
```

Comment:
Give the example of the provider init from the dev path

Reply:
Done